Source/Dafny/DafnyOptions.cs

```diff
@@ -255,11 +246,13 @@ public enum IncludesModes {
                   // There are no commas in paths, but there can be in arguments
                   var argumentsString = string.Join(',', pluginArray.Skip(1));
                   // Parse arguments, accepting and remove double quotes that isolate long arguments
-                  var splitter = new Regex(@"""((?:[^""]|\\"")*)""|([^ ]+)");
+                  var splitter = new Regex(@"""((?:[^""\\]|\\\\|\\"")*)""|([^ ]+)");
```

Comment:
Regular expressions with these many escaping always make me feel uncomfortable. 😬 
I'd suggest to extract this whole unescaping procedure into a separate method. Moreover, as I'm always in favor of expressive names I suggest using named capturing groups rather than numeric. Then it's easier to follow what you're actually trying to capture.

Reply:
I can't imagine what this regex would look like if we did not have the raw symbol "@" :-)